docs/src/stories/components/Layout/Stack.stories.jsx

```diff
@@ -0,0 +1,464 @@
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'stretch',
+        }
+      }
+    },
+    narrow_align: {
+      options: ['inherit', 'stretch', 'start', 'center', 'end', 'baseline'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit',
+        }
+      },
+    },
+
+    // Align wrap
+
+    alignWrap: {
+      options: ['start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets how stack lines are distributed, if they `wrap` into multiple lines. This has equivalent behavior to the `align-content` Flexbox property.',
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'start',
+        }
+      }
+    },
+
+    narrow_alignWrap: {
+      options: ['inherit', 'start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit',
+        }
+      },
+    },
+
+    // Spread
+
+    spread: {
+      options: ['start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio',
+      },
+      description: 'Sets how items will be distributed in the stacking direction.', 
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'start',
+        },
+      },
+    },
+
+    narrow_spread: {
+      options: ['inherit', 'start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio',
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Wrap
+
+    wrap: {
+      options: ['wrap', 'nowrap'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets whether items are forced onto one line or can wrap onto multiple lines.',
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'nowrap',
+        }
+      }
+    },
+    narrow_wrap: {
+      options: ['inherit', 'wrap', 'nowrap'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Divider
+
+    showDividers: {
+      control: {
+        type: 'boolean'
+      },
+      description: `Whether a divider between items is shown or not.
+
+_Note: the presence of a divider duplicates the \`gap\` between items._`,
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'false',
+        }
+      }
+    },
+    dividerAriaRole: {
```

Comment:
I'm not sure if we need this. It might be useful if we have a nested Stack. For example: an ActionList that stacks groups of menu items.

cc @ichelsea - is  an element with`role="separator"` the only way to divide groups of menu items? would it be weird if we used an element with `role="separator"` to divide list items or menu items? I'm worried about people accidentally misusing this role.

Reply:
@mperrotti I believe most `Stack` instances with dividers won't have to worry about this. By default dividers will have `role="presentation"`. But since `Stack` can be used for any situation, it might be good to give flexibility, since the `<hr />` element won't be available directly.

Reply:
@mperrotti groups of menu items within an ActionList should have their own aria-labels for group naming if I'm thinking of the right thing here. Support for grouped list items or menu items isn't super great at the moment, but the best we have is a screen reader user hearing that they're entering a group as they arrow through. All of that documentation should be in the [ADR for action components](https://github.com/github/primer/discussions/734).

Reply:
The only guidance I see for divider lines is for separating groups of menu items. It says to set `role="presentation"` and `aria-hidden="true"` ([example](https://user-images.githubusercontent.com/7228610/155743208-080fd521-27b1-4787-bc96-9c2e219c3b04.png)). This pretty much makes it invisible to screenreaders.

@ichelsea - Should we do the same thing to these dividers? Or do you think there are cases where we might want `role="separator"`?